regression-test/data/query/show/test_array_show_create.out

```diff
@@ -1,4 +1,4 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select --
-test_array_show_create	CREATE TABLE `test_array_show_create` (\n  `k1` int(11) NULL,\n  `k2` array<smallint(6)> NOT NULL,\n  `k3` array<int(11)> NOT NULL,\n  `k4` array<bigint(20)> NOT NULL,\n  `k5` array<char(1)> NOT NULL,\n  `k6` array<varchar(20)> NULL,\n  `k7` array<date> NOT NULL,\n  `k8` array<datetime> NOT NULL,\n  `k9` array<float> NOT NULL,\n  `k10` array<double> NOT NULL,\n  `k11` array<decimal(20, 6)> NULL\n) ENGINE=OLAP\nDUPLICATE KEY(`k1`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`k1`) BUCKETS 1\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
+test_array_show_create	CREATE TABLE IF NOT EXISTS `test_array_show_create` (\n  `k1` int(11) NULL,\n  `k2` array<smallint(6)> NOT NULL,\n  `k3` array<int(11)> NOT NULL,\n  `k4` array<bigint(20)> NOT NULL,\n  `k5` array<char(1)> NOT NULL,\n  `k6` array<varchar(20)> NULL,\n  `k7` array<date> NOT NULL,\n  `k8` array<datetime> NOT NULL,\n  `k9` array<float> NOT NULL,\n  `k10` array<double> NOT NULL,\n  `k11` array<decimal(20, 6)> NULL\n) ENGINE=OLAP\nDUPLICATE KEY(`k1`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`k1`) BUCKETS 1\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
```

Comment:
even create table with “IF NOT EXISTS”，which will not be displayed when show create table， u should move “IF NOT EXISTS”